docs/upgrade-to-v5.md

```diff
@@ -242,9 +244,9 @@ BSON.deserialize(result)
 ### Exports and available bundles
 
 Most users should be unaffected by these changes, Node.js `require()` / Node.js `import` will fetch the corresponding BSON library as expected.
-And for folks using bundlers like, webpack or rollup a tree shakable es module bundle will be pulled in because of the settings in our package.json.
+And for folks using bundlers like webpack or rollup, a tree shakable ES module bundle will be pulled in because of the settings in our `package.json`.
 
-Our package.json defines the following `"exports"` settings.
+Our `package.json` defines the following `"exports"` settings.
 ```json
 {
```

Comment:
This just needs an update to be accurate, it changed slightly after that PR:

```json
"main": "./lib/bson.cjs",
  "module": "./lib/bson.mjs",
  "exports": {
    "import": "./lib/bson.mjs",
    "require": "./lib/bson.cjs",
    "react-native": "./lib/bson.cjs",
    "browser": "./lib/bson.mjs"
  },
  ```

Reply:
Updated